plugins.json

```diff
@@ -3300,6 +3300,101 @@
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.4.9",
+    "hosting": "on-prem",
+    "author_type": "mattermost",
+    "release_stage": "beta",
+    "enterprise": false,
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmJyxTAACgkQ0bVLR6XO/sSZlw//QZ5AO3LbPigM9MWFQoCOWFjRbXWEhl4LxaMPEUsoWoB6TuHN0/0FKAUSNlP9fDIVifjn5THt8YFVXMnSWqVvgNgMPXRA+1DKUqMHFc5YdypiGoB8Qt4IXjoAgq75ZbZqTaOdyqamUb+bLBR/OdU3IU0CVTkNUdDEOz9r6VRTSmBaxtZIEWdsXtO1VsM2yo/UfXYcfLsqY0WUEfJsmVf7uiLXMTs3B6VlDyYldqu3FwfPpL/nFg+WxWJTw3sMUZI+g5wITxYFGgC0TNf5jnrLLxXo2t4TzwxFUKiOTbF0qN+RAYpO9xP1Rj6l7Asq9KzjnNdgg/26w4jRlyl3iIWRJAoPEuYvu+qZ7jU5wicxNuWWHvFMpItDcuUJZhoZdgTTJ3/dKi7sCQ9BGIxpRfW7CaW/5IyspQgIbxKBuv4+y05Kp/DEh7wrsDZ1173Hvs2FP6hMLK1CCjb5wFtHeqTgtwb1quA7wqDUKoHMuO7TT53CIl89GsxsHolG2qd0mBpu78+lMnA2XF5Zia6gJWCoTLjhKzbnFIhscKd9Lcw+ejR+vwS4FznljS+6D+Euy7v6x/SWR9iXpz4zgFPuddn5oslWrozBa+WK/zFLDpQVOhRjaB2UOaTYovK55xGZdadlInrnDBpW9tzqAmt7nOmW0l1/xneLdSliyxGOgfCnL4w=",
+    "repo_name": "mattermost-plugin-calls",
+    "manifest": {
+      "id": "com.mattermost.calls",
+      "name": "Calls",
+      "description": "Integrates real-time voice communication in Mattermost",
+      "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+      "support_url": "https://github.com/mattermost/mattermost-plugin-calls/issues",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.4.9",
+      "version": "0.4.9",
+      "min_server_version": "6.3.0",
+      "server": {
+        "executables": {
+          "linux-amd64": "server/dist/plugin-linux-amd64",
+          "darwin-amd64": "server/dist/plugin-darwin-amd64"
+        },
+        "executable": ""
+      },
+      "webapp": {
+        "bundle_path": "webapp/dist/main.js"
+      },
+      "settings_schema": {
+        "header": "",
+        "footer": "",
+        "settings": [
+          {
+            "key": "ICEHostOverride",
+            "display_name": "ICE Host Override",
+            "type": "text",
+            "help_text": "The IP (or hostname) to be used as the host ICE candidate. If empty, it defaults to resolving via STUN.",
+            "placeholder": "",
+            "default": ""
+          },
+          {
+            "key": "UDPServerPort",
+            "display_name": "RTC Server Port",
+            "type": "number",
+            "help_text": "The UDP port the RTC server will listen on.",
+            "placeholder": "8443",
+            "default": 8443
+          },
+          {
+            "key": "RTCDServiceURL",
+            "display_name": "RTCD Service URL",
+            "type": "text",
+            "help_text": "The URL to a running RTCD service instance that should host the calls. When set (non empty) all calls will be handled by the external service.",
+            "placeholder": "https://rtcd.example.com",
+            "default": null
+          },
```

Comment:
This makes me think a little as we are most likely not going to officially support `rtcd` on prem for the beta launch. But I guess there isn't a clean way to hide certain settings when not in Cloud?

Reply:
@streamer45 I don't think we have that ability -- we could add some explanation there if you're worried about it?

Reply:
Yeah, no worries. I think it's okay for the beta as supporting `rtcd` in on-prem is more about properly documenting everything at this point. Implementation wise it would work already so it's good to have it in sooner rather than later.